dolphinscheduler-dao/src/main/resources/org/apache/dolphinscheduler/dao/mapper/ErrorCommandMapper.xml

```diff
@@ -33,4 +33,26 @@
         </if>
         group by cmd.command_type
     </select>
+
+    <select id="queryErrorCommandPage" resultType="org.apache.dolphinscheduler.dao.entity.ErrorCommand">
+        select
+        *
+        from t_ds_error_command
+        where 1 = 1
+        order by process_instance_priority, id asc
+    </select>
+
+    <select id="queryErrorCommandPageByIds" resultType="org.apache.dolphinscheduler.dao.entity.ErrorCommand">
+        select
+        *
+        from t_ds_error_command
+        where 1 = 1
+        <if test="definitionCodes != null and definitionCodes.size() > 0">
+            and process_definition_code in
+            <foreach item="id" index="index" collection="definitionCodes" open="(" separator="," close=")">
+                #{id}
+            </foreach>
+        </if>
```

Comment:
```suggestion
        where process_definition_code in
            <foreach item="id" index="index" collection="definitionCodes" open="(" separator="," close=")">
                #{id}
            </foreach>
```
If the workflowDefinitionCodes is a required param, we would better not check it is empty here, if someone call this method but forget to set the workflowDefinitionCodes, it will get all commands.